config/jobs/kubernetes-sigs/sig-windows/soak-tests.yaml

```diff
@@ -1,62 +1,61 @@
 ---
+presets:
+- labels:
+    preset-azure-win-soak-test-creds: "true"
+  env:
+  - name: KUBECONFIG
+    value: "/root/.kube/win-soak-test"
+  volumeMounts:
+  - name: win-soak-test
+    mountPath: /root/.kube/win-soak-test
+  volumes:
+  - name: win-soak-test
+    secret:
+      secretName: win-soak-test
+      defaultMode: 420
+      items:
+      - key: win-soak-test
+        path: win-soak-test.kubeconfig
 presubmits:
-  kubernetes/perf-tests:
+  kubernetes/test-infra:
```

Comment:
Was k/test-infra targeted to make it easier to iterate on this PR?

Reply:
wondering the same thing... I think it was useful to have this job running on perf-test repo when making changes there.

Reply:
Oh I see, This moved the job so run just the e2e tests.  I believe we would want this to be a sperate job and to make this a periodic not a Pre-submit

Reply:
this way we can test changes and have a separate periodic job that runs against soak cluster

Reply:
This still looks like we moved this to Presubmit to `kubernetes/test-infra` repository.  Shouldn't it stay on `kubernetes/perf-tests` so we can test that any changes there can create a test?  

With the changes here it assumes we have a soak cluster and only triggers against the `kubernetes/perf-tests` which is not what we want I think.

Reply:
Yes, I changed this to test-infra to be able to trigger presubmit jobs while adjusting the parameters for these new jobs that are added. Going to switch back to perf-tests